README.md

```diff
@@ -12,7 +12,7 @@ Read the [documentation](https://docs.reach.sh).
 If you want to work on the Reach compiler, you'll need:
 - stack v2.7.1
 - z3 v4.8.10
-- solidity v0.8.5
+- solidity v0.8.7
```

Comment:
We should upgrade, but we actually don't require this one. We really do use v0.8.5 right now: https://github.com/reach-sh/reach-lang/blob/master/hs/Dockerfile#L13

If you were to use a different one, it is possible to get bad results.

I'll flag this for updating.

Reply:
ok ill change the pr, the snap install is the issue